widgetssdk/src/main/java/com/glia/widgets/call/CallController.java

```diff
@@ -227,7 +227,7 @@ public void initCall(String companyName,
         addOperatorMediaStateListenerUseCase.execute(operatorMediaStateListener);
         disposable.add(
                 gliaQueueForMediaEngagementUseCase
-                        .execute(queueId, contextUrl, mediaType)
```

Comment:
Is it okay to drop support for contextUrl now in this public  function?

Reply:
This class only for internal use.